debezium-connector-mysql/src/test/resources/ddl/decimal_column_test.sql

```diff
@@ -0,0 +1,12 @@
+-- ----------------------------------------------------------------------------------------------------------------
+-- DATABASE:  decimal_column_test
+-- ----------------------------------------------------------------------------------------------------------------
+
+CREATE TABLE dbz_751_decimal_column_test (
+      id INT AUTO_INCREMENT NOT NULL,
+      rating1 DECIMAL,
+      rating2 DECIMAL(8,4),
+      PRIMARY KEY (id)
+) DEFAULT CHARSET=utf8;
```

Comment:
Could you please also verify cases `DECIMAL(8)` and `DECIMAL(8,0)` and all cases also for `NUMERIC` datatype?

Reply:
@gunnarmorling It is fine now, thanks. Could you please add also NUMERIC, NUMERIC(8, 4) etc.? They should be handled in the same way but I've already seen a difference in their handling in the past so I'd say better say than sorry.